app/pages/docs/$$.mjs

```diff
@@ -31,6 +31,11 @@ export default function ({ html, state }) {
         border-radius: 8px;
         background-color: var(--grey-greyer);
       }
+
+      doc-video {
+        width: 100%;
+        aspect-ratio: 16 / 9;
```

Comment:
Technically, `<doc-video>` contains the player + the "Powered by `<mux-player>`" copy in the `<cite>` element, which takes up about 20px of vertical space, so this aspect-ratio isn't exactly right, but it's close